.github/workflows/main.yml

```diff
@@ -52,6 +52,8 @@ jobs:
             - name: Upload coverage report
               if: ${{ matrix.platform.os == 'iOS' }}
               uses: codecov/codecov-action@ab904c41d6ece82784817410c45d8b8c02684457
+              with:
+                token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
If the job uses a secret, we used to require an 'authorize' check to avoid running the job on third party PRs –so as to avoid leaking the secrets: https://github.com/auth0/auth0-flutter/blob/main/.github/workflows/main.yml#L175

I'd suggest checking with the team how you prefer to handle the usage of CI secrets going forward.

Reply:
This 'authorize' check that relies on a set of 'internal' and 'external' environments is described here: https://oktawiki.atlassian.net/wiki/spaces/DXSDK/pages/2814965005/Securing+GitHub+Actions

Reply:
Got it, will add `authorize` step for now and will discuss internally within the team as well on handling secrets going forward.